recipes/caffe/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Setup CMake build location
+mkdir build
+cd build
+
+# Configure, build, test, and install.
+cmake -DCPU_ONLY=1 -DBLAS="open" -DCMAKE_INSTALL_PREFIX="${PREFIX}" ..
+make
+make runtest
+make install
+
+# Python installation is non-standard. So, we're fixing it.
+mv "${PREFIX}/python/caffe" "${SP_DIR}/"
+find "${PREFIX}/python/" -name "*.py" | xargs chmod +x
+find "${PREFIX}/python/" -name "*.py" | xargs -I {} cp {} "${PREFIX}/bin/"
```

Comment:
Pretty gross that these still have `.py` extensions, but are intended to be binaries. Maybe we could strip them?